amqp-common/src/main/java/uk/nhs/adaptors/amqp/task/TaskType.java

```diff
@@ -0,0 +1,17 @@
+package uk.nhs.adaptors.amqp.task;
+
+import lombok.Getter;
+
+public enum TaskType {
```

Comment:
I'm not a fan of any "Test code" in production code. No one will ever remove it if we keep it.